src/components/Backtester/reports/HistoricalReport.js

```diff
@@ -1,53 +1,79 @@
+// eslint-disable
```

Comment:
Maybe we should only disable some specific ESLint rules, but not the ESLint itself here?

Reply:
removed as it was added for local testing purpose only